bricksrc/entity_properties.py

```diff
@@ -0,0 +1,244 @@
+from .namespaces import BRICK, TAG, OWL, RDFS, SKOS, UNIT, XSD
+from rdflib import Namespace, Literal
+
+# these are the "relationship"/predicates/OWL properties that
+# relate a Brick entity to a structured value.
+# These are all instances of Brick.EntityProperty, which is
+# a subclass of OWL.ObjectProperty
+entity_properties = {
+    BRICK.hasArea: {
```

Comment:
I know we briefly discussed this before without a deeper discussion, but now it concerns me a bit, so I think better late than never: 

I feel `has` is redundant to all the entity property names here. When it comes to a noun as a property name, it's quite obvious that all the objects would be a type of property for the subject. Not having `has` is a common practice across many data models. E.g., [Microsoft graph API](https://docs.microsoft.com/en-us/graph/api/resources/contact?view=graph-rest-1.0#properties) and [Haystack](https://project-haystack.org/tag/area).

(@corymosiman12 raised this concern too, let me know if you'd like to add anything here)